examples/Buttons/BorderlessButton.md

```diff
@@ -79,6 +79,51 @@ import { faTrash } from '@fortawesome/free-solid-svg-icons';
   }}
   variant={VARIANT.DANGER}
 >
-  <FontAwesomeIcon icon={faTrash} size="small" />
+  <FontAwesomeIcon icon={faTrash} size="lg" />
 </BorderlessButton>
 ```
+
+### With the `forwardRef`
+Ref example: The optional `forwardRef` property is set. When the primary themed button is clicked, the focus shifts to the borderless button.
+```jsx
+import {
+  useState,
+  useRef,
+} from 'react';
+import {
+  Button,
+  VARIANT,
+  } from 'mark-one';
+import { FontAwesomeIcon } from '@fortawesome/react-fontawesome';
+import { faTrash } from '@fortawesome/free-solid-svg-icons';
+
+const BorderlessButtonRefExample = () => {
+  const ref = useRef(null);
+  const [value, setValue] = useState('');
+  const onButtonClick = () => {
+    setTimeout(() => ref.current.focus());
+  }
```

Comment:
Why not   `const onButtonClick = () => {ref.current.focus()}`?

Reply:
Ah yes, you're right that the `setTimeout` is not necessary - thanks for catching that! I removed this through 8721dbb.